### PR TITLE
[graviton][tensorflow] update ecr allow list for patching

### PR DESCRIPTION
**Pull request by xncqr**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Updating the ECR allow lists in support of Patching TensorFlow 2.9.1 Graviton

### Tests run
[fdc680f](https://github.com/aws/deep-learning-containers/pull/2685/commits/fdc680f8fad13e00f25fa840cd32e68b70568476), [1c590ef](https://github.com/aws/deep-learning-containers/pull/2685/commits/1c590efce9b92d17cce72bd44a1e8effc91fe56c)
* EC2 Test
* ECS Test
* EKS Test
* Sanity Test

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kenny-ezirim — APPROVED** [review]

File: tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json [hunk 1]
```diff
@@ -0,0 +1,974 @@
...
+      "cvss_v2_score": 6.8,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32547.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32547 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " In ImageMagick, there is load of misaligned address for type 'double', which requires 8 byte alignment and for type 'float', which requires 4 byte alignment at MagickCore/property.c. Whenever crafted or untrusted input is processed by ImageMagick, this causes a negative impact to application availability or other problems related to undefined behavior.",
```
Comment (kenny-ezirim) [inline tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json:29]: The json record here and the one above seem to be the same and missing reason_to_ignore. Any reason for the record duplication and omission of the key "reason_to_ignore"?

File: tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json [hunk 2]
```diff
@@ -0,0 +1,974 @@
...
+      "cvss_v2_score": 6.8,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32545.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32545 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " A vulnerability was found in ImageMagick, causing an outside the range of representable values of type 'unsigned char' at coders/psd.c, when crafted or untrusted input is processed. This leads to a negative impact to application availability or other problems related to undefined behavior.",
```
Comment (kenny-ezirim) [inline tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json:77]: Same record duplication and omission of reason_to_ignore observed. Please review allowlist.json.

File: tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json (hunk 1 above)
Comment (xncqr) [reply tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json:29]: Only doing for the package and not every as we cannot put next to the package due to the json structure, this is only provided to the top most CVE so it is clearly seen for the package. Any more and we are just making unneeded noise and work as it's the package that needs to be updated.

File: tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json (hunk 2 above)
Comment (xncqr) [reply tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json:77]: same response as above.

**Pull request merged**